README.md

```diff
@@ -511,6 +511,7 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 
 
 ### Ruby
+- [Resmi Ruby web sitesi](https://www.ruby-lang.org/tr/)
```

Comment:
:)

Reply:
En basta gorunce linke hic bakmamisim. Pot mi kirdim dersin Gokay hocam?

Reply:
Evet, en sona koymalıyız da resmi web sitesi en başı hak etmiyor mu? =)

Reply:
Bence "en sona koyma kuralı" biraz esnek olmalı.

Reply:
@fatihacet to the rescue